lib/rubocop/cop/mixin/array_min_size.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Common functionality for handling array count minimums
```

Comment:
I'd extend this documentation a bit, perhaps referencing the use-cases warranting the existence of the mixin. At the very least `array count minimums` sounds pretty vague and confusing to me.